src/components/Composer/index.android.js

```diff
@@ -102,7 +102,10 @@ class Composer extends React.Component {
                 onContentSizeChange={(e) => ComposerUtils.updateNumberOfLines(this.props, e)}
                 rejectResponderTermination={false}
                 textAlignVertical="center"
-                maximumNumberOfLines={!this.props.isComposerFullSize ? this.props.maxLines : undefined}
+                // Setting a really high number here fixes an issue with the react-native fork, where on Android the text input would collapse to only one line,
+                // when it should actually expand to the container (https://github.com/Expensify/App/issues/11694#issuecomment-1560520670)
+                // TODO: @Szymon20000 is working on fixing this (android-only) issue in the react-native fork, remove this commend once done
```

Comment:
As of a few days ago we agreed we're not going to use the fork anymore. Can you please include the link to the upstream PR?

Reply:
Updated the comment and added the link to the [upstream PR](https://github.com/facebook/react-native/pull/35703)